tutorial/markdown/java/spring-boot.md

```diff
@@ -53,172 +91,494 @@ mvn package
+This command will execute all the test cases in your project.
 
-```java
-final List<Profile> profiles = cluster.query("SELECT p.* FROM $bucketName._default.$collectionName p WHERE lower(p.firstName) LIKE $search OR lower(p.lastName) LIKE $search LIMIT $limit OFFSET $skip",
-  queryOptions().parameters(JsonObject.create()
-  .put("bucketName", dbProperties.getBucketName())
-  .put("collectionName", CollectionNames.PROFILE)
-  .put("search", "%"+ search.toLowerCase()+"%")
-  .put("limit", limit)
-  .put("skip", skip))
-  .scanConsistency(QueryScanConsistency.REQUEST_PLUS))
-  .rowsAs(Profile.class);
-return ResponseEntity.status(HttpStatus.OK).body(profiles);
+### Run Individual Tests:
+
+Additionally, you can run individual test classes or methods using the following commands:
+
+To run the tests for the AirlineIntegrationTest class:
+
+```sh
+mvn test -Dtest=org.couchbase.quickstart.springboot.controllers.AirlineIntegrationTest
 ```
 
-> *from getProfiles method of controllers/ProfileController.java*
+To run the tests for the AirportIntegrationTest class:
 
-### Running The Tests
+```sh
+mvn test -Dtest=org.couchbase.quickstart.springboot.controllers.AirportIntegrationTest
+```
 
-To run the standard integration tests, use the following commands:
+To run the tests for the RouteIntegrationTest class:
 
-```shell
-mvn test
+```sh
+mvn test -Dtest=org.couchbase.quickstart.springboot.controllers.RouteIntegrationTest
 ```
 
-### Project Setup Notes
+## Project Setup Notes
 
-This project was based on the standard [Spring Boot project](https://spring.io/guides/gs/rest-service/).  The HealthCheckController is provided as a santity check and is used in unit tests.
+This project was based on the standard [Spring Boot project](https://spring.io/guides/gs/rest-service/). The HealthCheckController is provided as a santity check and is used in unit tests.
 
 A full list of packages are referenced in the pom.xml file.
 
-## Conclusion
+## Contributing
+
+Contributions are welcome! If you'd like to contribute to this project, please fork the repository and create a pull request.
+
+## Appendix
+
+### Extending API by Adding New Entity
+
+If you would like to add another entity to the APIs, these are the steps to follow:
+
+- Create the new entity (collection) in the Couchbase bucket. You can create the collection using the [SDK](https://docs.couchbase.com/java-sdk/current/howtos/provisioning-cluster-resources.html#collection-management) or via the [Couchbase Server interface](https://docs.couchbase.com/cloud/n1ql/n1ql-language-reference/createcollection.html).
+- Define the routes in a new class in the `controllers` package similar to the existing routes like `AirportController.java`.
+- Define the service in a new class in the `services` package similar to the existing services like `AirportService.java`.
+- Define the repository in a new class in the `repositories` package similar to the existing repositories like `AirportRepository.java`.
```

Comment:
You explained Airline artifacts. Why do you introduce a new Airport artifact now?

Also mention about tests

Reply:
This is for extending a new entity. Since Airline is already present i didnt mention airline itself.
Mentioned about the tests